src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.test.tsx

```diff
@@ -21,10 +21,10 @@ import React from 'react';
 import { StepIndexPattern } from '../step_index_pattern';
 import { shallowWithI18nProvider } from 'test_utils/enzyme_helpers';
 import { Header } from './components/header';
-import { IndexPatternCreationConfig } from '../../../../../../../../../../plugins/index_pattern_management/public';
-import { coreMock } from '../../../../../../../../../../core/public/mocks';
-import { dataPluginMock } from '../../../../../../../../../../plugins/data/public/mocks';
-import { SavedObjectsFindResponsePublic } from '../../../../../../../../../../core/public';
+import { IndexPatternCreationConfig } from '../../../../../../../plugins/index_pattern_management/public';
+import { coreMock } from '../../../../../../../core/public/mocks';
+import { dataPluginMock } from '../../../../../../../plugins/data/public/mocks';
+import { SavedObjectsFindResponsePublic } from 'src/core/public';
```

Comment:
nit: maybe at least in one file we will use one pattern for imports. will look neater
```
import { coreMock } from '../../../../../../../core/public/mocks';
or
import { SavedObjectsFindResponsePublic } from 'src/core/public';
```